pom.xml

```diff
@@ -664,6 +653,12 @@
         <artifactId>slf4j-api</artifactId>
         <version>${slf4j.version}</version>
       </dependency>
+      <dependency>
+        <!-- version specified to converge transitive dependency of hadoop and curator -->
+        <groupId>org.xerial.snappy</groupId>
+        <artifactId>snappy-java</artifactId>
+        <version>1.1.8.4</version>
```

Comment:
Weird. Looks like they included a java port of Snappy.

Reply:
Yes, Hadoop recently moved away from using the native libraries directly for snappy and lz4. They use Java dependencies now, which may include their own native implementation.